### PR TITLE
migrated all usages of models.MetadataUser & models.User classes to g…

### PR DESCRIPTION
**Pull request by omriAl**
…enerated.schema.teams.user.User classes instead.

### Describe your changes :
migrated all usages of models.MetadataUser & models.User classes to generated.schema.teams.user.User classes instead. I This was in order to use only the native pydantic generated models.

#
### Type of change :

- [] Bug fix
- [x] Improvement
- [] New feature
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

@amiorin @pmbrull

### CODE REVIEW

**Comment by harshach** [comment]
Thanks for taking this up @omriAl

**Review by harshach — APPROVED** [review]

**Comment by harshach** [comment]
@omriAl looks like check style is breaking.
```
make install_dev
make precommit_install
```

re-commit these changes in your branch so that isort and black can run and it will fix these issues

File: ingestion/tests/integration/ldap/test_ldap_crud.py [hunk 1]
```diff
@@ -17,16 +17,17 @@
 import time
 from datetime import datetime
 
-
 import pytest
 import requests
 from ldap3 import ALL, Connection, Server
 
-from metadata.ingestion.models.user import MetadataUser, User
+# from metadata.ingestion.models.user import MetadataUser, User
```
Comment (mithmatt) [inline ingestion/tests/integration/ldap/test_ldap_crud.py:24]: Remove this commented out import?

**Comment by pmbrull** [comment]
> @omriAl looks like check style is breaking.
> 
> ```
> make install_dev
> make precommit_install
> ```
> 
> re-commit these changes in your branch so that isort and black can run and it will fix these issues

Thanks, @omriAl for tackling this! The docs on the `ingestion` setup have not yet been published to the gitbook so that is why you might have missed them, sorry for that.

As @harshach says, installing the recommit hooks will take care of formatting code & imports while you develop, as well as running `flake8` to pick up linting issues early on.

Another option is to directly call `make isort black`, which will run the tools on demand.

Hope this helps!

**Comment by amiorin** [comment]
@harshach 
Is it safe to merge this PR?
image

**Review by harshach — CHANGES_REQUESTED** [review]
@omriAl looks like you included the commits that shouldn't be in this PR. Please checkout a new branch and cherry-pick the changes you made and create a new PR.

**Comment by omriAl** [comment]
@harshach yes, my bad, my main branch on the fork was behind.
created a new one:
https://github.com/open-metadata/OpenMetadata/pull/1473

**Comment by mithmatt** [comment]
Closing out of date PR, actual changes covered in #1473

**Pull request closed without merging**